spec/dynamoid/persistence_spec.rb

```diff
@@ -681,12 +680,12 @@ def reload_address
 
     it 'uses dumped value of sort key to call UpdateItem' do
       klass = new_class do
-        range :activated_at, :datetime
+        range :activated_on, :date
         field :name
       end
       klass.create_table
 
-      obj = klass.create!(activated_at: Time.now, name: 'Old value')
+      obj = klass.create!(activated_on: Date.today, name: 'Old value')
```

Comment:
`Date.today` is dangerous and should generally be avoided as it is not Rails time zone aware.  Especially in specs where we might have people copying the patterns in their own code.  Use `Time.zone.today` instead.

Ref: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/Date

Reply:
@pboling Completely agree with you. In Rails application we should use `Time.zone.now` and `Time.zone.today` (and their synonyms).

But it isn't a Rails application. Dynamoid doesn't rely on `Rails.config.timezone`. It could use it 
 (as default value for `application_timezone`) in order to simplify integration with Rails.

IMHO in specs it doesn't matter and it only couples Dynamoid with Rails even more, but Dynamod can be used apart from Rails.

Reply:
I'm ok with that reasoning 👍